### DISCUSSION TITLE
Dark mode prefixing doesn't work in classes

### DISCUSSION THREAD

**Discussion by robin901118** [tailwindlabs/tailwindcss#7208] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.0.16

**What build tool (or framework if it abstracts the build tool) are you using?**

vite

**What version of Node.js are you using?**

v16.6.2

**Reproduction URL**

[https://play.tailwindcss.com/D7zdzNsb31](https://play.tailwindcss.com/D7zdzNsb31)

**Describe your issue**
In the case of adding a prefix, the dark mode is set to a class, and the toggle has no effect.
But it works in the case of media queries.
If you remove the prefix, it can also take effect

**Comment by adamwathan** [comment, marked as answer]
Hey! The `dark` class you are adding to the root of the document needs the prefix too.

**Reply by robin901118** [reply]
all right! thanks
